lib/views/create_project_form.dart

```diff
@@ -33,7 +33,7 @@ class _CreateProjectFormState extends State<CreateProjectForm> {
         children: [
           fields(context),
           LoadingButton(
-            text: "Create my first project",
+            text: "Create my project",
```

Comment:
We can add a parameter to the widget as a bool and leave "Create my first project" for the first project?

Reply:
In my opinion this adds complexity to the code for something that does not really change the user experience. 
@taorepoara @shiipou Do you have a preference?